Sources/MenuAnimator.swift

```diff
@@ -18,8 +18,14 @@
 
 import UIKit
 
+/**
+ Options of menu transitioning.
+ */
```

Comment:
I think it would be good to see full parameter list which can be customized. So, let's use the first option with a full list.